pkg/openscap/openscap.go

```diff
@@ -190,7 +191,7 @@ func (s *defaultOSCAPScanner) oscapChroot(oscapArgs ...string) ([]byte, error) {
 	return out, err
 }
 
-func (s *defaultOSCAPScanner) Scan(mountPath string, image *docker.Image) ([]iiapi.Result, interface{}, error) {
+func (s *defaultOSCAPScanner) Scan(ctx context.Context, mountPath string, image *docker.Image, filter iiapi.FilesFilter) ([]iiapi.Result, interface{}, error) {
```

Comment:
`filter` here is unused which makes me think it shouldn't be part of the interface.  Is there a nice way to make this a Scanner specific option without making every scanner implement it?

Reply:
@pweil- Yes. Stop using interface.

Reply:
heh, that's not a "nice" way :smile: 

I guess the relevant question here is what does this have to do with usage of the context like this PR is designed to do?  `FileFilter` seems unrelated.

Reply:
@pweil- `FileFilter` was removed from this PR.